shared/teams/team/container.js

```diff
@@ -128,7 +158,11 @@ export default compose(
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
   lifecycle({
     componentDidMount: function() {
-      this.props._loadTeam(this.props.name)
+      const teamname = this.props.name
```

Comment:
Note that now we're loading details for all ancestor teams, and not just the current team.